Shared/Samples/Manage operational layers/ManageOperationalLayersView.swift

```diff
@@ -0,0 +1,155 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import SwiftUI
+import ArcGIS
+
+struct ManageOperationalLayersView: View {
+    /// A map with a topographic basemap and centered on western USA.
+    @State private var map = {
+        let map = Map(basemapStyle: .arcGISTopographic)
+        map.initialViewpoint = Viewpoint(
+            center: Point(x: -133e5, y: 45e5, spatialReference: .webMercator),
+            scale: 2e7
+        )
+        return map
+    }()
+    
+    /// A Boolean value indicating whether to show the manage layers sheet.
+    @State private var isShowingSheet = false
+    
+    /// A Boolean value indicating whether to show an alert.
+    @State private var isShowingAlert = false
+    
+    /// The error shown in the alert.
+    @State private var error: Error? {
+        didSet { isShowingAlert = error != nil }
+    }
+    
+    var body: some View {
+        MapView(map: map)
+            .task {
+                do {
+                    // Add layers from urls.
+                    let elevationImageLayer = ArcGISMapImageLayer(url: .worldElevations)
+                    try await elevationImageLayer.load()
+                    
+                    let censusTiledLayer = ArcGISMapImageLayer(url: .censusTiles)
+                    try await censusTiledLayer.load()
+                    
+                    map.addOperationalLayers([elevationImageLayer, censusTiledLayer])
+                } catch {
+                    self.error = error
+                }
+            }
+            .toolbar {
+                ToolbarItem(placement: .bottomBar) {
+                    Button("Manage Layers") {
+                        isShowingSheet = true
+                    }
+                    .sheet(isPresented: $isShowingSheet, detents: [.medium], dragIndicatorVisibility: .visible) {
+                        NavigationView {
+                            ManageLayersSheetView(map: $map)
+                                .navigationTitle("Manage Layers")
+                                .navigationBarTitleDisplayMode(.inline)
+                        }
+                    }
+                }
+            }
+            .alert(isPresented: $isShowingAlert, presentingError: error)
+    }
+}
+
+struct ManageLayersSheetView: View {
+    /// The map with the operational layers.
+    @Binding var map: Map
```

Comment:
```suggestion
    let map: Map
```
The map can also be a constant in this specific sample.
I think making it a binding is good thinking, in case the map is swapped by the superview, so no need to change.

Reply:
If the map is swapped by the superview, its body will be recomputed and this view will be re-created. The only reason to use a binding would be if this view needed to _set_ the map. If it only _reads_ the map, `map` should be immutable.